webpack/base.config.js

```diff
@@ -92,6 +95,7 @@ module.exports = (env, argv) => ({
                         loader: 'file-loader',
                         options: {
                             name: 'app/helptexts/[name].[ext]',
+                            esModule: false,
```

Comment:
What does this `esModule` field do?

Reply:
That's a new field in `file-loader` version `5.x`. It's by default set to `true`, but for some reason it has to be `false` in our settings. See https://github.com/webpack-contrib/file-loader#esmodule and https://github.com/webpack-contrib/file-loader/issues/350#issuecomment-558260039.